screwdriver/screwdriver.go

```diff
@@ -8,10 +8,13 @@ import (
 	"net/http"
 	"net/url"
 	"path"
+	"regexp"
 	"strconv"
 	"strings"
 )
 
+var regexpForUnmarshal = regexp.MustCompile(`"(.*?)" *: *"(.*?)"`)
```

Comment:
Do we need white space before `*` ?
(I don't know what kind of string will be come.)

Reply:
Basically, we do not need the white spaces. However we need them in test because the jsons we are using for testing contain whitespace.